create_package.py

```diff
@@ -394,39 +458,73 @@ def main(
         help=(
             "Skip zipping server package and create only"
             " server folder structure."
-        )
+        ),
     )
     parser.add_argument(
         "--keep-sources",
         dest="keep_sources",
         action="store_true",
-        help=(
-            "Keep folder structure when server package is created."
-        )
-    )
-    parser.add_argument(
-        "-c", "--clear-output-dir",
-        dest="clear_output_dir",
-        action="store_true",
-        help=(
-            "Clear output directory before package creation."
-        )
+        help="Keep folder structure when server package is created.",
     )
-
     parser.add_argument(
-        "-o", "--output",
+        "-o",
+        "--output",
         dest="output_dir",
         default=None,
         help=(
             "Directory path where package will be created"
             " (Will be purged if already exists!)"
-        )
+        ),
+    )
+    parser.add_argument(
+        "--only-client",
+        dest="only_client",
+        action="store_true",
+        help=(
+            "Extract only client code. This is useful for development."
+            " Requires '-o', '--output' argument to be filled."
+        ),
+    )
+    parser.add_argument(
+        "--debug",
+        dest="debug",
+        action="store_true",
+        help="Debug log messages."
+    )
+    parser.add_argument(
+        "--upload",
+        dest="upload",
+        action="store_true",
+        help="Upload the build to your ayon server and reload",
     )
 
     args = parser.parse_args(sys.argv[1:])
-    main(
-        args.output_dir,
-        args.skip_zip,
-        args.keep_sources,
-        args.clear_output_dir
-    )
+    upload_package = args.upload and not args.skip_zip
+    if upload_package and not has_ayon_api:
+        raise RuntimeError(
+            "Module 'ayon_api' is not available. Please install it"
+            " to use the upload feature (pip install ayon-python-api)."
+        )
+
+    level = logging.INFO
+    if args.debug:
+        level = logging.DEBUG
+    logging.basicConfig(level=level)
+
+    output_dir: str = get_output_dir(args.output_dir)
+
+    main(output_dir, args.skip_zip, args.keep_sources, args.only_client)
+    if upload_package:
```

Comment:
Do we really need upload logic?

Reply:
Personally, I believe it should exist some where for reference. 
And, since this is an example then I believe we can leave that logic here for reference.